Assets/Editor/ModelInspector.cs

```diff
@@ -1,8 +1,14 @@
 using UnityEngine;
-using System.Reflection;
+using UnityEngine.Rendering;
 using System;
+using System.Collections.Generic;
+using System.IO;
```

Comment:
`System.IO`, `System.Text`, `UnityEditor.SceneManagement`, `Object = System.Object` are all additions that are not needed. Rider shows that, use that info! 
![Clipboard01](https://user-images.githubusercontent.com/348087/77400201-794ec380-6db3-11ea-9b4e-0a7cc269aa01.png)